infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -297,6 +297,7 @@ public void reset() {
     @Override
     public void reboot() {
```

Comment:
do we need to turn this into synchronized? otherwise, what if there is reboot between issuing tokens on global log and streams (or between multiple streams)?